### PR TITLE
BZ2004045: Rescanning creates four additional mc

### PR DESCRIPTION
**Pull request by xenolinux**
Rescanning creates four additional mc for profile
xccdf_org.ssgproject.content_profile_moderate

OCP version: 4.6, 4.7, 4.8, 4.9, 4.10
[BZ2004045](https://bugzilla.redhat.com/show_bug.cgi?id=2004045)
[Preview](https://deploy-preview-39673--osdocs.netlify.app/openshift-enterprise/latest/security/compliance_operator/compliance-operator-advanced#compliance-rescan_compliance-advanced)
[preview2](https://deploy-preview-39673--osdocs.netlify.app/openshift-enterprise/latest/security/compliance_operator/compliance-operator-remediation.html#compliance-review_compliance-remediation)
QA contact: @sunilcio

### CODE REVIEW

File: modules/compliance-rescan.adoc [hunk 1]
```diff
@@ -11,6 +11,22 @@ Typically you will want to re-run a scan on a defined schedule, like every Monda
 $ oc annotate compliancescans/<scan_name> compliance.openshift.io/rescan=
 ----
 
+After rescan, four mc will be generated.
```
Comment (JAORMX) [inline modules/compliance-rescan.adoc:14]: This does not happen for all profiles.  It's only applicable to the rhcos-moderate profile.

**Review by JAORMX — CHANGES_REQUESTED** [review]

File: modules/compliance-rescan.adoc (hunk 1 above)
Comment (xenolinux) [reply modules/compliance-rescan.adoc:14]: Should I explicitly mention there as such?

Comment (JAORMX) [reply modules/compliance-rescan.adoc:14]: Yes, that would be ideal.

Comment (xenolinux) [reply modules/compliance-rescan.adoc]: ok. Changed the sentence

File: modules/compliance-rescan.adoc [hunk 2]
```diff
@@ -11,6 +11,22 @@ Typically you will want to re-run a scan on a defined schedule, like every Monda
 $ oc annotate compliancescans/<scan_name> compliance.openshift.io/rescan=
 ----
 
+A rescan generates four `mcp` for `rhcos-moderate` profile:
```
[the remark this answers is not in the document]
Comment (xenolinux) [reply modules/compliance-rescan.adoc]: addressed

File: modules/compliance-rescan.adoc [hunk 3]
```diff
@@ -11,6 +11,22 @@ Typically you will want to re-run a scan on a defined schedule, like every Monda
...
+
+[source,terminal]
+----
+$ oc get mcp
```
[the remark this answers is not in the document]
Comment (xenolinux) [reply modules/compliance-rescan.adoc]: addressed

**Review by lpettyjo — COMMENTED** [review]
LGTM

**Pull request merged**